### PR TITLE
Release 174 0

### PR DESCRIPTION
**Pull request by bijayprasadtiwari**

### CODE REVIEW

File: docs/cmdlets-reference/get-cohesityphysicalagentstatus.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Get-CohesityPhysicalAgentStatus
```
Comment (pyashish) [inline docs/cmdlets-reference/get-cohesityphysicalagentstatus.md:1]: can't this be : Get-CohesityAgentStatus ?

Comment (pyashish) [reply docs/cmdlets-reference/get-cohesityphysicalagentstatus.md:1]: 4 words seem too long for a cmdlet

File: src/Cohesity.Powershell/Scripts/ProtectionSource/Get-CohesityPhysicalAgentStatus.ps1 [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    )
+
+    Begin {
+    }
+
+    Process {
+            $cohesityUrl = '/irisservices/api/v1/public/protectionSources/registrationInfo?includeEntityPermissionInfo=true&includeApplicationsTreeInfo=false&allUnderHierarchy=true&environments=kPhysical'
+            $resp = Invoke-RestApi -Method Get -Uri $cohesityUrl
+            if (200 -eq $Global:CohesityAPIStatus.StatusCode) {
+                $nodes = $resp.rootNodes
```
Comment (pyashish) [inline src/Cohesity.Powershell/Scripts/ProtectionSource/Get-CohesityPhysicalAgentStatus.ps1:27]: Does these parse through nested objects or there are none when using the above filters ?

File: docs/cmdlets-reference/get-cohesityphysicalagentstatus.md (hunk 1 above)
Comment (bijayprasadtiwari) [reply docs/cmdlets-reference/get-cohesityphysicalagentstatus.md:1]: Renamed the cmdlet to Get-CohesityAgentStatus

File: src/Cohesity.Powershell/Scripts/ProtectionSource/Get-CohesityPhysicalAgentStatus.ps1 (hunk 2 above)
Comment (bijayprasadtiwari) [reply src/Cohesity.Powershell/Scripts/ProtectionSource/Get-CohesityPhysicalAgentStatus.ps1:27]: The nodes are flat objects and no nested nodes.

**Pull request merged**